src/main/scala/Bintray.scala

```diff
@@ -60,11 +60,11 @@ object Bintray {
     BintrayCredentials.read(credsFile).fold(sys.error(_), _ match {
       case None =>
         if (prompt) {
-          println("bintray-sbt requires your bintray credentials.")
+          log.info("bintray-sbt requires your bintray credentials.")
```

Comment:
We actually want to use the sbt.Logger. Can you add a parameter to `def ensuredCredentials` called `log: Logger`? Then in BintrayPlugin.scala:99 add an argument `streams.value.log` to pass the instance of the logger?